tests-hivemq-platform-operator/build.gradle.kts

```diff
@@ -4,7 +4,7 @@ plugins {
 
 group = "com.hivemq.helmcharts"
 
-val testPlanOtherExcludeTags = "Containers,CustomConfig,Extensions,Platform,PodSecurityContext,ServiceAccount,Services,Upgrade,Volumes"
+val testPlanOtherExcludeTags = "Containers,ClusterDomainName,CustomConfig,Extensions,Platform,PodSecurityContext,ServiceAccount,Services,Upgrade,Volumes"
```

Comment:
Just double checked with the Gradle task which tests are executed in `Other`:
```
Searching for tests in package 'com.hivemq.helmcharts' (includeTags: none) (excludeTags: Containers,ClusterDomainName,CustomConfig,Extensions,Platform,PodSecurityContext,ServiceAccount,Services,Upgrade,Volumes)
Found 7 integration tests:
  com.hivemq.helmcharts.single.HelmManifestsIT
  com.hivemq.helmcharts.single.HelmApplyCrdIT
  com.hivemq.helmcharts.compose.HelmDataHubIT
  com.hivemq.helmcharts.single.HelmInstallRemoteImagesIT
  com.hivemq.helmcharts.compose.HelmRemoveNamespaceIT
  com.hivemq.helmcharts.compose.HelmLegacyStatefulSetMigrationIT
  com.hivemq.helmcharts.single.HelmLicenseSecretIT
```
Looks good to me!